docs/GUIDES.md

```diff
@@ -73,7 +73,118 @@ The playing field is a grid of tiles, addressable like a coordinate system.
    │ 0│ 1│ 2│ 3│ 4│ 5│ 6│ 7│ 8│ 9│10│11│12│13│14│15│16│17│18│19│20│
 </pre>
 
-## Writing Code
+## Tutorial
+
+### 0. Prerequisites
+
```

Comment:
it may sound silly, but I think we should have a `explain what snake is`

Reply:
Good point :+1: 
I'm not sure if this part belongs to the `Prerequisites` section, I would  rather add it to an introduction part at the beginning which explains:
- what it is the scope of the game
- How it works
- etc.

What do you think?

Maybe we can also find a better name than `prerequisites` 🤔 like: `Before starting` or similar

Reply:
Should we create an issue for tracking that? 
This "introduction" part is not strictly related to the first chapter, so I would not tackle it right now but in another PR.

Reply:
👍 to everything said here 🙇 

Outlining the "goal" or "product" of the workshop is something I'm also missing from the [rails girls tutorial](http://guides.railsgirls.com/app). We should definitely have that (and it would be the place to explain what snake is).

I would definitely want to a picture like this then as well 🎉 
![image](https://user-images.githubusercontent.com/3491815/68298925-e24ecc80-009a-11ea-89cb-6ea0b594e2a4.png)

I initially thought we'd have a *"concepts"* section in the beginning for explaining everything. But reading your comments, I'm changing my mind: it's too dry and too much unnecessary information for getting started with the tutorial itself. Maybe it makes more sense, to spread this conceptual information more though the tutorial and put it closer to where it applies.

So I'd suggest to have an *"About"* section in the beginning, where we give a rough outline of what we will be doing (including what snake is). *"Prerequisites*" would then in my opinion still be a separate next section to prepare for the actual hands on coding.

What do you think?

Reply:
Sounds like a good approach to me

![agreed](https://media.giphy.com/media/iU8395XCj6ltC/giphy.gif)

Reply:
Ok, moved this into https://github.com/codecurious-bln/elixir_snake/issues/10 and will tackle the section in a separate PR.

Fire for you?